dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/pushpublish/PushPublishFilterResourceTest.java

```diff
@@ -0,0 +1,81 @@
+package com.dotcms.rest.api.v1.pushpublish;
+
+import com.dotcms.mock.request.MockAttributeRequest;
+import com.dotcms.mock.request.MockHeaderRequest;
+import com.dotcms.mock.request.MockHttpRequest;
+import com.dotcms.mock.request.MockSessionRequest;
+import com.dotcms.mock.response.MockHttpResponse;
+import com.dotcms.publishing.FilterDescriptor;
+import com.dotcms.rest.ResponseEntityView;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.exception.DotDataException;
+import com.google.common.collect.ImmutableMap;
+import java.util.Map;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+import org.glassfish.jersey.internal.util.Base64;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import com.dotcms.rest.exception.SecurityException;
+
+public class PushPublishFilterResourceTest {
+
+    static HttpServletResponse response;
+    static PushPublishFilterResource resource;
+    static String filterKey;
+
+    @BeforeClass
+    public static void prepare() throws Exception {
+        // Setting web app environment
+        IntegrationTestInitService.getInstance().init();
+        resource = new PushPublishFilterResource();
+        response = new MockHttpResponse();
+
+        APILocator.getPublisherAPI().getFilterDescriptorMap().clear();
```

Comment:
I suggest that the internal map is not exposed but offer the methods you need via API (e.g. clear())

Reply:
done